articles/healthcare-apis/iot/iot-connector-overview.md

```diff
@@ -30,11 +33,14 @@ Below is an overview of each step IoT connector does once IoMT device data is re
 
 ## Scalable
 
-IoT connector is designed out-of-the-box to handle even the most demanding IoMT device message workloads by using autoscaling features. 
+IoT connector is designed out-of-the-box to support growth and adaptation to the changes and pace of healthcare by using autoscaling features. The service enables developers to modify and extend the capabilities to support additional device mapping template types and FHIR resources.
 
 ## Configurable 
 
-IoT connector is configured by using [Device](./how-to-use-device-mappings.md) and [FHIR destination](./how-to-use-fhir-mappings.md) mappings. The mappings instruct the filtering and transformation of your IoMT device messages into the FHIR format.
+IoT connector is configured by using Device and FHIR destination mappings. The mappings instruct the filtering and transformation of your IoMT device messages into the FHIR format.
+The different points for extension are:
+•	Normalization: Health data from disparate devices can be aligned and standardized into a common format to make sense of the data from a unified lens and capture trends.
+•	FHIR Conversion: Health data is normalized and grouped by mapping commonalities to FHIR. Observations can be created or updated according to chosen or configured templates. Devices and health care consumers can be linked for enhanced insights and trend capture.
```

Comment:
Should we use a lowercase "c" for conversion?

Reply:
yes.